concent_api/gatekeeper/tests/test_gatekeeper_views.py

```diff
@@ -124,6 +124,237 @@ def test_upload_should_return_401_if_file_paths_are_not_unique(self):
         self.assertIn('message', response.json().keys())
         self.assertEqual("application/json", response["Content-Type"])
 
+    @freeze_time("2018-12-30 11:00:00")
```

Comment:
All these tests are very repetitive. Could you replace them with a two tests (one for checksums, one for file sizes) where each has a list of valid and invalid values and goes over them in a loop?